### ISSUE TITLE
Base ontology url behaving oddly

### ISSUE THREAD

**Issue by radmilak** [EBISPOT/efo#658]
Hi, I noticed the url 'http://www.ebi.ac.uk/efo/efo.owl' started behaving weird lately - in the browser the ontology file never starts to download, when trying to download from a python script or with wget, the download starts but gets stalled/extremely slow after a while. The alternative url ('https://github.com/EBISPOT/efo/releases/download/current/efo.owl') works without any problem, but for our application it is quite beneficial if the download url is the same as the ontology base url.

Thanks!

**Comment by paolaroncaglia** [comment]
@radmilak 
Thanks for reporting this. The efo.owl file is 163 MB, so downloading it will take a couple of minutes anyway. This said, I was able to replicate your issue. I'm not sure what the cause is, so I'll ask my colleagues @zoependlington or @simonjupp . Please bear with us as they are both at a work meeting today. Meanwhile, may I please ask what application you are using EFO for? It'd be helpful for us to know. Thanks!

**Comment by radmilak** [comment]
Thanks for a quick reply! Yes I am aware the file is quite large and waiting a couple of minutes is no problem. In short our project tries to create a standard for metadata related to genomic annotation track files (https://fairtracks.github.io/). The standard is described in a form of json schemes, and some of the items are prescribed to be terms in specific ontologies, including EFO.  We can then use this information when processing the metadata to get for example term labels from the ontologies. We also include version information (versionIRI) about the used ontologies as a part of the metadata files, and especially for this it is quite practical that the base url matches the one we used to download the file, as this is true for the other ontologies we use.

**Comment by paolaroncaglia** [comment]
@radmilak 
Thanks for the details! We'll get back to you next week about the base URL issue.
Best wishes,
Paola

**Comment by zoependlington** [comment]
Hi @radmilak, thank you for creating this ticket. My colleague (@warrenread) and I have tested downloading EFO through wget and cURL and have not been able to replicate your problem here.

@warrenread mentioned:
> Both “curl -L”  and plain “wget” work for me on http://www.ebi.ac.uk/efo/efo.owl , whether from macOS or Linux. “curl” on its own doesn’t work, as there’s no redirect, but wget seems to do this by default. wget is at v1.30 on Linux and 1.20 on macOS.

When did you notice the wget expiring/downloading slowly etc.? It could be that there was a brief problem at our end that caused a problem in downloading and that it is now rectified. However, if you are still having this issue, please let us know so we can investigate further. Thanks!

**Comment by radmilak** [comment]
Hi, thanks for the answer! I first noticed the problem last Wednesday (11.12.) and was able to reproduce it several times Wednesday-Friday (I waited a bit before posting to be sure). I experienced the problem both when downloading from python (with urllib) and when trying manually with wget (haven't tried curl). 
That said, I haven't been able to reproduce the issue today, so perhaps it was as you are saying, that this was some temporary problem that is resolved now. Thanks!
